backend/model/model.go

```diff
@@ -634,7 +634,7 @@ type Session struct {
 	VerboseID             string  `json:"verbose_id"`
 
 	// Excluded will be true when we would typically have deleted the session
-	Excluded *bool `gorm:"default:false"`
```

Comment:
We don't need this to be a pointer since the default value is false (both at the database level and with the Go zero value). By not using a pointer, it makes the code way more readable, see below ⬇️